### PR TITLE
Fixed #22774 Can't update totals for credit note

### LINKED ISSUE #22774: Can't update totals for credit note
### Preconditions (*)
1. Magento Open Source 2.3.x & 2.3-develop(after 2.3.3 code was merged)

### Steps to reproduce (*)
1. Select an invoice
2. Click "Credit Memo"
3. Add value to `Refund Shipping `, `Adjustment Refund` or `Adjustment Fee` field

### Expected result
1. Should be able to update totals using the "Update Totals" button

M2 2 8

### Actual result
1. There is no way to update the total (without changing the quantity and then changing it back again)

M2 3 1

### PR DESCRIPTION
**Pull request by gaurav-473**
Fixed #22774 Can't update totals for credit note
### Preconditions (*)
1. Magento Open Source 2.3.1

### Steps to reproduce (*)
1. Select an invoice
2. Click "Credit Memo"
3. Add value to `Refund Shipping `, `Adjustment Refund` or `Adjustment Fee` field

### Expected result
1. Should be able to update totals using the "Update Totals" button

M2 2 8

### Actual result
1. There is no way to update the total (without changing the quantity and then changing it back again)

M2 3 1

### CODE REVIEW

**Comment by swnsma** [comment]
Hi @guarinogabriel,
Thank you for your contribution!

As I can see, you are trying to add changes which had been done in 2.2-develop with commit 451f7b9f317db5a27d3d9c541cc3c1678cc169f5 in scope of issue #11798 and these changes were not Forwardported to 2.3-develop.

May I ask you to create PR with all changes from the 451f7b9f317db5a27d3d9c541cc3c1678cc169f5 ?

**Review by swnsma — CHANGES_REQUESTED** [review]
Hi @guarinogabriel,
Thank you for your contribution!

As I can see, you are trying to add changes which had been done in 2.2-develop with commit 451f7b9 in scope of issue #11798 and these changes were not Forwardported to 2.3-develop.

May I ask you to create PR with all changes from the 451f7b9 ?

**Comment by gaurav-473** [comment]
Hi @swnsma,
I updated all changes of commit 451f7b9.

File: app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php [hunk 1]
```diff
@@ -4,9 +4,7 @@
  * See COPYING.txt for license details.
  */
 namespace Magento\Sales\Block\Adminhtml\Order\Creditmemo\Create;
-
```
Comment (swnsma) [inline app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php]: Can we avoid such changes?

File: app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/totals/adjustments.phtml [hunk 2]
```diff
@@ -39,7 +40,6 @@
                    id="adjustment_negative"/>
             <script>
                 require(['prototype'], function(){
-
```
Comment (swnsma) [inline app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/totals/adjustments.phtml:42]: Can we avoid such changes?

**Review by swnsma — CHANGES_REQUESTED** [review]
Hi @gauravagarwal1001 
Thank you for your changes.

Could you please:
1) Dismiss remove of such amount empty lines
2) Check and adjust changes according to static tests build fail.
3) Check and adjust failed functional tests.

File: app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php (hunk 1 above)
Comment (gauravagarwal1001) [reply app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php]: Sure I removed empty lines.

File: app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/totals/adjustments.phtml (hunk 2 above)
Comment (gauravagarwal1001) [reply app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/totals/adjustments.phtml]: Sure I removed empty lines

File: app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php (hunk 1 above)
Comment (gaurav-473) [reply app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php]: There must be one blank line after the namespace declaration.

**Comment by swnsma** [comment]
@gauravagarwal1001,  I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**